experimental/cmake/sst.cmake.back

```diff
@@ -0,0 +1,247 @@
+# cmake/sst.cmake
```

Comment:
This is my mistake, but if we are happy with sst.cmake we should delete this file.